.github/workflows/pilotWrapper.yml

```
@@ -0,0 +1,23 @@
+name: pilot wrapper
+
+on: [push, pull_request]
+
+jobs:
+  PilotWrapper:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: False
+      matrix:
+        python:
+          - 2.7.13
+          - 3.6.8
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: prepare environment
+      run: conda create -n python_${{ matrix.python }} python=${{ matrix.python }}
+    - name: run pilot wrapper test
+      run: |
```

I think you're not activating the environment?
```suggestion
      run: |
        conda activate python_${{ matrix.python }}
```

---

ouf! Thanks.

I'll commit this and your other suggestions.